frontend/packages/console-shared/src/components/pod/PodRing.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
We should fix the problem instead of ignoring. We definitely shouldn't ignore at the file level.

Reply:
It wants me to add `clickCount` to the dependency array which I don't think is necessary, I don't want to execute `useEffect` when clickCount changes, just `obj.spec.replicas`.  I can more the ignore to line-level

Reply:
fixed, moved ignore to line-level.

Reply:
@dtaylor113 whenever we disable rules, sometimes it's obvious, but when it's not please add a comment otherwise someone else will come in and fix the lint error.

Reply:
Hi @christianvogt, added eslint comment.  -thanks